features/search-replace.feature

```diff
@@ -15,6 +15,12 @@ Feature: Do global search/replace
       guid
       """
 
+    When I run `wp search-replace foo bar --include-columns=post_content`
+    Then STDOUT should not contain:
+      """
+      guid
+      """
```

Comment:
Can you make this test a bit more explicit? Given `guid` is skipped by default, I don't think this will ever fail.